.github/workflows/release.yaml

```diff
@@ -6,30 +6,22 @@ on:
 
 env:
   FORCE_COLOR: 2
-  NODE: 14.x
+  NODE: 16
 
 jobs:
   release:
-    if: "!contains(github.event.head_commit.message, 'skip ci') && github.repository == 'tancredi/fantasticon'"
```

Comment:
This is no longer needed: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/